lib/chain_spec/configs/mainnet.ex

```diff
@@ -2,9 +2,5 @@ defmodule MainnetConfig do
   @moduledoc """
   Mainnet config constants.
   """
-
-  @parsed_config ConfigUtils.load_config_from_file!("config/networks/mainnet/config.yaml")
-  @unified Map.merge(MainnetPreset.get_preset(), @parsed_config)
-
-  def get(key), do: Map.fetch!(@unified, key)
+  use ChainSpec.GenConfig, file: "config/networks/mainnet/config.yaml", preset: MainnetPreset
```

Comment:
The preset is a value in the config, can't we fetch it from config and parse it instead of having to select it here?

Reply:
Yeah, we can

Reply:
I added it in 1d6063b